projects/components/src/summary-list/summary-list.component.test.ts

```diff
@@ -0,0 +1,32 @@
+import { SummaryListComponent } from '@hypertrace/components';
+import { createComponentFactory, Spectator } from '@ngneat/spectator/jest';
+
+describe('Summary List component', () => {
+  let spectator: Spectator<SummaryListComponent>;
+
+  const createComponent = createComponentFactory({
+    component: SummaryListComponent,
```

Comment:
For a component with inputs and ouputs, using a hostfactory allows you to provide inputs and then verify it rendered correctly. This test only verifies the logic directly rather than for a given set of inputs it renders the html we want - that's what we're generally looking for in a component's tests - the functions are implementation details.